### PR TITLE
Create force-thick-frames-plus.wh.cpp

### PR DESCRIPTION
**Pull request by teknixstuff2**
Thick Framer but better

### CODE REVIEW

File: mods/force-thick-frames-plus.wh.cpp [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            canMod = true;
+        }
+    }
+
+    HWND ret = NtUserCreateWindowEx_Original(
+        dwExStyle, UnsafeClassName, VersionedClass, UnsafeWindowName, dwStyle,
+        x, y, nWidth, nHeight, hWndParent, hMenu, hInstance, lpParam,
+        dwShowMode, dwUnknown1, dwUnknown2, qwUnknown3);
+    if (canMod && ret != NULL) {
+        WindhawkUtils::SetWindowSubclassFromAnyThread(ret, &antiResizeProc, 0);
```
Comment (m417z) [inline mods/force-thick-frames-plus.wh.cpp:118]: You don't remove the subclass on uninit, which probably means that processes that have the subclass set will just crash.

Comment (teknixstuff2) [reply mods/force-thick-frames-plus.wh.cpp:118]: Whoops. And you're right. I'm just not sure how to go about fixing that.

Comment (m417z) [reply mods/force-thick-frames-plus.wh.cpp:118]: Look at one of the mods that use `SetWindowSubclassFromAnyThread`, for example `msg-box-font-fix`:

https://github.com/ramensoftware/windhawk-mods/blob/16c1c93b12f2e2fb707c419efb19a7d3c3b80a11/mods/msg-box-font-fix.wh.cpp#L70

https://github.com/ramensoftware/windhawk-mods/blob/16c1c93b12f2e2fb707c419efb19a7d3c3b80a11/mods/msg-box-font-fix.wh.cpp#L214-L217

https://github.com/ramensoftware/windhawk-mods/blob/16c1c93b12f2e2fb707c419efb19a7d3c3b80a11/mods/msg-box-font-fix.wh.cpp#L177-L186

https://github.com/ramensoftware/windhawk-mods/blob/16c1c93b12f2e2fb707c419efb19a7d3c3b80a11/mods/msg-box-font-fix.wh.cpp#L307-L317

**Comment by MiRw3b** [comment]
seems cool but what are the changes xddd

**Comment by MiRw3b** [comment]
update: i had to disable it because it caused STEAM to not work. (like it bootlooped the steam webhelper service)..

**Comment by ArbestRi02** [comment]
Any more updates?

**Comment by teknixstuff** [comment]
This mod wasn't ever finished and is buggy. If you're using openglass, there's a better alternative mod available from https://github.com/Maplespe/DWMBlurGlass/issues/147#issuecomment-2052520952 (this mod doesn't work correctly with DWMBlurGlass however, despite being on the DWMBlurGlass repo)

**Pull request closed without merging**